src/lib/netlist/solver/nld_ms_gcr.h

```diff
@@ -161,7 +161,7 @@ namespace netlist::solver
 
 			//# auto gtot_t = std::accumulate(gt, gt + term_count, plib::constants<FT>::zero());
 			//# *tcr_r[railstart] = static_cast<FT>(gtot_t); //mat.A[mat.diag[k]] += gtot_t;
-			auto pd = this->m_mat_ptr[k][net.rail_start()] - &this->mat.A[0];
+			std::size_t pd =std::size_t(this->m_mat_ptr[k][net.rail_start()] - &this->mat.A[0]);
```

Comment:
This line lost the space after the assignment operator `=`.

Reply:
Fixed